aten/src/ATen/native/Cross.cpp

```diff
@@ -37,28 +69,13 @@ Tensor & cross_out(const Tensor & input, const Tensor & other, const c10::option
   return at::linalg_cross_out(out, input, other, dim);
 }
 
-Tensor linalg_cross(const Tensor & input, const Tensor & other, const int64_t dimension) {
-  Tensor out = at::empty({0}, input.options());
-  native::linalg_cross_out(input, other, dimension, out);
-  return out;
-}
-
-Tensor & linalg_cross_out(const Tensor & input, const Tensor & other, const int64_t dimension, Tensor & out) {
-  // Broadcast inputs
+TORCH_IMPL_FUNC(linalg_cross_out)
+(const Tensor & input, const Tensor & other, const int64_t dim, const Tensor & out) {
   auto out_size = infer_size(input.sizes(), other.sizes());
   Tensor input_broadcasted = input.expand(out_size);
   Tensor other_broadcasted = other.expand(out_size);
```

Comment:
This is redundant b/w meta and impl function. I'm not aware of a way to avoid this, is this OK for now?